dev/git.ts

```diff
@@ -171,10 +171,24 @@ const gitGenerators: Record<string, Fig.Generator> = {
       }
 
       const files = output.split("\n").map((file) => {
+        let alreadyAdded = false;
+        // From "git status --short"
+        // M  dev/github.ts // test file that was added
+        //  M dev/kubectl.ts // test file that was not added
+        // A  test2.txt // new added and tracked file
+        // ?? test.txt // new untracked file
+        if (file.charAt(0) === "M" || file.charAt(0) === "A") {
+          alreadyAdded = true;
+        }
```

Comment:
We can simplify since by assigning directly `alreadyAdded`:
```suggestion
        // From "git status --short"
        // M  dev/github.ts // test file that was added
        //  M dev/kubectl.ts // test file that was not added
        // A  test2.txt // new added and tracked file
        // ?? test.txt // new untracked file
        const alreadyAdded = file.charAt(0) === "M" || file.charAt(0) === "A";
```